File: src/dataset/javascript/isogram_exercism-elenamountz/claude/ClaudeSonnet4_isogram_v1.js
export const isIsogram = (str) => {
  const seen = new Set();
  for (let i = 0; i < str.length; i++) {
    const char = str[i].toLowerCase();
    if (char >= 'a' && char <= 'z') {
      if (seen.has(char)) {
        return false;
      }
      seen.add(char);
    }
  }
  return true;
};

File: src/dataset/c/isogram_exercism-EzequielMarcelo/claude/ClaudeSonnet4_isogram_v1.c
#include "isogram.h"

bool is_isogram(const char phrase[]) 
{
    if (phrase == NULL) 
        return false;

    uint32_t seen = 0;

    for (const char *p = phrase; *p != '\0'; ++p) 
    {
        if ((*p >= 'A' && *p <= 'Z') || (*p >= 'a' && *p <= 'z')) 
        {
            int bit_pos = (*p & 0x1F) - 1;
            
            if (seen & (1U << bit_pos)) 
                return false;
        
            seen |= (1U << bit_pos);
        }
    }

    return true;
}